README.md

```diff
@@ -92,6 +92,12 @@ and `DRAINPIPE_PROCESSOR`. Valid platform values are `linux`, `darwin`, or `wind
 and processors are `386`, `amd64`, or `arm64`. These correspond to builds of
 upstream dependencies e.g. https://github.com/go-task/task/releases
 
+### Node JS
+
+All Drainpipe components (DDEV, Github Actions, Tugboat) are configured to use
+the same Node JS major version. This is set in the `.nvmrc` file, in the Drainpipe
+project root.
+
```

Comment:
Does it mean that the user can create a file in the docroot or project root to override this?
How will the user control the node version?

Reply:
Sure! Thanks @rabbitlair!
If this file is now scaffolded in the project, will it be overwritten when I update it? I will test that.
If users can use and override that file, we need to update the docs in this PR.

Reply:
The idea is the file gets scaffolded only if it does not previously exist. So, if users overwrite it, Drainpipe will not scaffold it again.

Reply:
Awesome, thank you! Hopefully, all tests will pass soon, and we should be able to approve this one.

Reply:
And you remove the `nodejs_version` property from `.ddev/config.yaml` ?
That's interesting.